actionpack/test/controller/content_type_test.rb

```diff
@@ -92,14 +92,14 @@ def test_content_type_from_render
   # :ported:
   def test_charset_from_body
     get :render_charset_from_body
-    assert_equal Mime::HTML, @response.content_type
+    assert_equal Mime::TEXT, @response.content_type
     assert_equal "utf-16", @response.charset
   end
 
   # :ported:
   def test_nil_charset_from_body
     get :render_nil_charset_from_body
-    assert_equal Mime::HTML, @response.content_type
+    assert_equal Mime::TEXT, @response.content_type
     assert_equal "utf-8", @response.charset, @response.headers.inspect
   end
 
```

Comment:
We should match shipped behavior at this point. I was thinking the idea for `render body: '…'` was to *not* set the Content-Type at all, though?